### DISCUSSION TITLE
Zertifikate Dritter verlinken

### DISCUSSION THREAD

**Discussion by Paul-B98** [Bootstrap-Academy/Bootstrap-Academy#3] [Ideas]
So lange auf der Plattform keine Bildungszertifikate erworben werden können und über den Zeitraum Hinaus wäre es interessant welche Zertifizierungsmöglichkeiten zu einem erarbeiteten Thema bestehen.

Eine Möglichkeit wäre zu den Reitern Kurse, Coachings und Webinare noch einen Reiter Zertifikate einzufügen auf dem diese Informationen bereitgestellt werden. Dieser kann entsprechend auh später genutzt werden eigene Zertifikate Anzubieten sollte diese Möglichkeit bestehen.

**Comment by Cameo007** [comment]
[Open Badges](https://de.m.wikipedia.org/wiki/Open_Badges)  wären eine Möglichkeit.

**Reply by Cameo007** [reply]
In den Metadaten sind Infos zum Zertifikat enthalten und auch eine Verifizierungs-URL.

[openHPI](https://open.hpi.de) benutzt die zB.

**Comment by TheMorpheus407** [comment]
Es gibt derzeit den Issue, dass wir public profiles bauen. Ich würde das danach machen und dann für jeden einzelnen Skill ein Zertifikat auswählen, das wir supporten
